### PR TITLE
feat: show the components instead of describing them

### PR DESCRIPTION
**Pull request by shawn-sandy**
Implements [`docs/plans/add-homepage-design-direction.md`](docs/plans/add-homepage-design-direction.md). The homepage now renders live component specimens instead of describing them.

## Why

The design-token layer in `_design-tokens.scss` was decorative. `--card-background` and `--header-background` had **zero `var()` consumers**, so every card and the header band fell through to the `@fpkit/acss` defaults — including a hardcoded `whitesmoke` on the header. The dark-mode block compounded it by redefining `:root` variables and nothing else, so **no element repainted** and dark mode could not change a pixel. Meanwhile the homepage listed feature names as text, and the whole page rendered in a single font.

## What changed

**Tokens.** Seven direction tokens (`--ink`, `--ink-soft`, `--paper`, `--paper-sunk`, `--island`, `--island-bg`, `--rule`) flipping in both `prefers-color-scheme: dark` and the `:root[data-theme]` override path. The toggle path wins on specificity alone — `(0,2,0)` vs `(0,1,0)` — so block order doesn't matter and a chosen theme always beats the OS preference.

**"Colour marks hydration."** Astro withholds JavaScript, so the palette says what the framework says: static content is ink on paper, and `--island` appears only where something is interactive. An E2E test walks every element on five routes and fails if a non-interactive one computes to the accent.

**Three type roles** where there was one: display (self-hosted Space Grotesk 600, 13KB latin subset, preloaded, `font-display: swap`), body (system sans), mono (code and labels).

**`HomeHero.astro`** — a full-bleed band pairing a rendered `Card` with the import line that produces it. Content stays on the 80rem measure via `padding-inline: max(gutter, (100% - maxw) / 2)`, which spans the viewport while pushing content inward; at 320px the calculation goes negative and `max()` falls back to the gutter, so reflow is unaffected.

**`FeatureCards.astro`** gains `sectionTitle` and `promoted` for two-tier output. Both default to the existing six-card layout, so current call sites are untouched.

## Verification

Measured against this repo's dev server, not assumed.

| Plan target | Before | After |
|---|---|---|
| distinct font families | 1 | **3** |
| hero display-to-deck ratio | 1.63 | **3.20** at 320/390/768/1280 |
| `--card-background` consumers | 0 | **1** painting rule |
| `--header-background` consumers | 0 | **1** painting rule |
| dark rules painting elements | 0 | **3** per dark scope |
| non-interactive elements with the accent | — | **0** across 5 routes × 2 themes |

Reflow sweep: **48 combinations** (6 routes × 4 widths × 2 themes), **zero** overflow.

Gates vs. baseline: type-check **127 = 127** errors (unchanged, all pre-existing in `src/utils/*`); vitest **457 passing vs 437**, +20, zero new failures; e2e **50 passed / 2 failed**; `npm run build` exit 0, 65 pages; stylelint and prettier clean.

## Reviewer notes

**`eslint.config.js` gained an `e2e/**` override.** Playwright specs were falling through to the unit-test override, which declares no browser globals, so every `page.evaluate` callback using `getComputedStyle`, `Element` or `Node` failed `no-undef` at the pre-commit hook. This is a real gap — it would have bitten the next person to write an E2E test that inspects computed style — but it is adjacent to the feature, so flagging it rather than burying it.

**The two failing E2E tests are pre-existing and unrelated.** Both are `GET /@id/astro/runtime/client/dev-toolbar/entrypoint.js → 504 Outdated Optimize Dep` — Astro's own dev toolbar being re-optimized by Vite. Dev-server-only; the production build is clean. They survive a cold restart and no file here touches them.

**One acceptance criterion is ticked on a vacuous pass.** Criterion 8 names `/docs`, which returns a **pre-existing 500**: `getEntry('docs', '0-welcome')` at `src/pages/docs/index.astro:6` resolves `undefined` and the `!` assertion lets `render(undefined)` throw. The collection has held no `0-welcome` entry since before `52cde11`. Its zero-overflow reading is therefore vacuous rather than a genuine pass — noted inline in the plan rather than passed off as green. Tracked in the plan's Next Steps.

**The committed baseline file was corrected mid-flight.** Its first E2E recording (16 failed / 23 passed) did not measure this repository. `e2e/test-utils.ts:1` hardcodes `BASE_URL` to port 4321, bypassing `use.baseURL`, and `playwright.config.ts` sets `reuseExistingServer` — so with another Astro project holding 4321 the suite silently graded *that* app. Confirmed with `lsof`. The phantom failures were plausible enough to look like real regressions in this branch, including a 320px reflow failure. Corrected baseline against the right server: 2 failed / 50 passed. Also tracked in Next Steps.

**`CHANGELOG.md` marks an earlier entry as superseded.** A prior entry documents pinning `body` to a literal `#fff` specifically so the page stays white in both colour schemes. This PR reverses that deliberately — it is now `var(--paper)`. The old entry is kept as the record of why the literal existed, with a note that it is no longer current behaviour.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

- **New Features**
  - Added a redesigned homepage hero with responsive layouts, calls to action, accessibility guidance, and interactive card specimens.
  - Added promoted feature cards with tier labels, code examples, and responsive presentation.
  - Added light and dark themes with updated colors, typography, surfaces, focus states, and self-hosted display fonts.
  - Added a Design Direction guide to the documentation sidebar, including customization and accessibility guidance.
- **Documentation**
  - Documented the completed design direction, theme tokens, typography, component styling, and verification results.
- **Accessibility**
  - Improved keyboard focus visibility and color contrast across themes.

### CODE REVIEW

File: src/components/astro/HomeHero.astro [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+---
+
+<header class="home-hero" data-home-hero>
+  <div class="home-hero__lede">
+    <p class="home-hero__eyebrow" data-ui="eyebrow">{eyebrow}</p>
+    <h1 class="home-hero__display" data-ui="display">{headline}</h1>
+    <p class="home-hero__deck" data-home-hero-deck>{deck}</p>
+
+    <div class="home-hero__cta">
+      <a class="home-hero__button" href="/docs">Browse components</a>
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/components/astro/HomeHero.astro]: Fixed in ee120c9 — both the CTA and the specimen card now resolve through a single `specimenLink` constant pointing at `/guide/components/` (verified 200; `/docs` confirmed 500 locally). Left `/docs` itself alone as a pre-existing defect outside this plan. Added an e2e test that follows every hero link and asserts the destination responds under 400, and confirmed it fails with "/docs should not be an error page" when the CTA is regressed.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/styles/_design-tokens.scss [hunk 2]
```diff
@@ -224,39 +333,46 @@
...
+// preference. Starlight already stamps this attribute on the docs routes.
+:root[data-theme="dark"] {
+  @include direction-dark;
+  @include legacy-palette-dark;
+  @include direction-dark-surfaces;
+}
+
+:root[data-theme="light"] {
+  @include direction-light;
 }
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/styles/_design-tokens.scss:393]: Fixed in ee120c9 — extracted the light palette into a `legacy-palette-light` mixin and included it in the override, so both token families flip together as they do in the dark override. Verified under emulated OS-dark with `data-theme="light"`: `--color-neutral-200` and `--card-border` now resolve to `#e4e4e7` instead of staying `#3f3f46` against light `--paper`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: e2e/home-accessibility.spec.ts [hunk 3]
```diff
@@ -30,4 +30,192 @@ test.describe('Home Page Accessibility', () => {
...
+      // A ring counts if it paints: a non-zero outline, a box-shadow ring, or a
+      // colour change against the unfocused state.
+      const hasOutline = after!.outlineWidth > 0 && after!.outlineStyle !== 'none'
+      const hasShadow = after!.boxShadow !== 'none' && after!.boxShadow !== ''
+      const changedColor = after!.color !== before!.color
+
+      expect(
+        hasOutline || hasShadow || changedColor,
+        `${colorScheme}: focused ${after!.tag} shows no visible focus indicator (outline ${after!.outlineWidth}px ${after!.outlineStyle} ${after!.outlineColor}, box-shadow ${after!.boxShadow})`
+      ).toBe(true)
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply e2e/home-accessibility.spec.ts]: Fixed in 28f30f9 — both readings now come from the element Tab actually landed on (measured focused, blurred, re-measured), so the comparison can no longer be between two unrelated nodes.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: e2e/homepage-design-direction.spec.ts [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+    // respond, which is what the baseline failed to do.
+    const painted = [
+      ['body > header', light.header, dark.header],
+      ['[data-card]', light.card, dark.card],
+    ].filter(([, l, d]) => l !== null && d !== null && l !== d)
+
+    expect(
+      painted.length,
+      `no painted surface repainted. header ${light.header} -> ${dark.header}, card ${light.card} -> ${dark.card}`
+    ).toBeGreaterThanOrEqual(1)
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply e2e/homepage-design-direction.spec.ts:253]: Fixed in 28f30f9 — added explicit not.toBeNull() assertions on both probes in both schemes before the filter, so a selector that stops matching fails loudly instead of silently dropping that surface.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: e2e/homepage-design-direction.spec.ts [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+test.describe('Homepage design direction', () => {
+  // Review baseline: exactly ONE family (-apple-system) rendered across every
+  // h1, h2, h3, p, a and button, so headings and body copy differed only by
+  // size. The plan assigns three roles: display, body sans, mono.
+  test('three distinct font families resolve across heading, body and label', async ({ page }) => {
+    await page.setViewportSize({ width: 1280, height: 900 })
+    await page.goto('/')
+
+    const roles = await page.evaluate(() => {
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply e2e/homepage-design-direction.spec.ts:46]: Fixed in 28f30f9 — await document.fonts.ready after each goto, both before the family probe and inside the reflow width loop.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: e2e/home-responsive.spec.ts [hunk 6]
```diff
@@ -37,4 +37,41 @@ test.describe('Home Page Responsive Design', () => {
...
+          widest,
+        }
+      })
+
+      expect(
+        scrollWidth - clientWidth,
+        `${colorScheme}: scrollWidth ${scrollWidth} vs clientWidth ${clientWidth}; widest element ${widest}`
+      ).toBe(0)
+    })
+  }
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply e2e/home-responsive.spec.ts]: Fixed in 28f30f9 — deleted the single-scheme test (the parameterised light case is a strict superset: same page, same assertion, plus it names the widest offender) and relaxed toBe(0) to toBeLessThanOrEqual(0).

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: project-docs/03-features/design-direction.md [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+| `--ink-soft`   | `#5a6472` | `#99a2ad` | Secondary text: decks, eyebrows, labels, captions, inline code |
+| `--paper`      | `#fcfcfd` | `#0d1014` | Page background                                                |
+| `--paper-sunk` | `#f1f3f5` | `#151a20` | Recessed surfaces: the header band, card backgrounds           |
+| `--island`     | `#5b2cf5` | `#9b7dff` | The accent. Interactive only                                   |
+| `--island-bg`  | `#f0ebff` | `#1e1830` | Accent wash behind interactive affordances and `::selection`   |
+| `--rule`       | `#dde1e6` | `#262c33` | Hairlines and dividers                                         |
+
+The light values are deliberately not Tailwind sky-500 and not indigo-600, so the palette does not read as a framework
+default. Every pair clears WCAG 2.1 AA: in light, ink on paper is roughly 18:1, ink-soft on paper roughly 6.4:1, and
+island on paper roughly 6.7:1; in dark, every pair is 6.1:1 or better.
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply project-docs/03-features/design-direction.md]: Fixed in 28f30f9 — corrected to the measured 5.85:1 in project-docs, the Starlight guide and the token comment. Also corrected island/paper from 6.7:1 to 6.56:1, and added the lowest pair (ink-soft/paper-sunk, 5.39:1), which none of the three documents named.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: docs/plans/add-homepage-design-direction.md [hunk 8]
```diff
@@ -48,15 +49,15 @@ A clickable prototype of the target state, with a Current/Proposed toggle and a
...
+6. [x] Create `src/components/astro/HomeHero.astro` with a mono eyebrow, a left-aligned display headline, a deck at roughly a third of the headline size, two calls to action and a specimen figure pairing a rendered `Card` with the import line that produces it, forward the named slot through `src/layouts/Layout.astro` as `{Astro.slots.has('header') && <slot name="header" slot="header" />}`, then render `HomeHero` from `src/pages/index.astro` into that slot Why: `index.astro` renders `Layout.astro`, not `Base.astro`, and `Layout.astro` today forwards only `featured`, the default slot and the two sidebar slots, so a bare `slot="header"` on the page is silently dropped before it ever reaches the `Astro.slots.has('header')` branch at `src/layouts/Base.astro:102` — and the forward must stay guarded, because an unconditional `<slot name="header" slot="header" />` makes `slots.has('header')` true on every route and would suppress the default `Header` sitewide Verify: the homepage renders `HomeHero` at a display-to-deck font-size ratio of 3.0 or above with `text-align: left`, and `/about`, `/posts/1` and `/docs` each still render the original `Header` with its page title.
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply docs/plans/add-homepage-design-direction.md]: Fixed in 28f30f9 — step 6 now describes the fallback-slot implementation and records why the guarded forward does not work (Astro registers the forwarded slot key statically, so slots.has is true on every route and the sitewide header disappears). HTML regenerated with plan-agent-render.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: docs/plans/add-homepage-design-direction.md [hunk 9]
```diff
@@ -68,17 +69,17 @@ Tier 1 — This plan changes application code
...
+- [x] No non-interactive element computes to the accent colour — 0 offenders across `/`, `/posts/1`, `/content/1`, `/tags` and `/about` in both themes, scanning 88-134 elements per route with interactive elements and their descendants excluded
+- [x] Routes other than `/` still render the shared `Header` component — `/about`, `/posts/1`, `/tags`, `/blog` and `/content/1` each render `<header>` with their own page title and no `HomeHero` markup. The forward supplies the default `Header` as slot fallback content rather than using an `Astro.slots.has()` guard, because a forwarded named slot registers statically and would otherwise suppress the header sitewide
+- [x] The direction is documented in `project-docs/` and in the Starlight guide — `project-docs/03-features/design-direction.md` and `src/content/docs/guide/design-direction.mdx`, wired into the Starlight `Features` sidebar. Note the guide path is `guide/` (singular), not the `guides/` this plan's Files section names; the collection has always been `src/content/docs/guide/` and every sidebar link resolves under `/guide/`
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply docs/plans/add-homepage-design-direction.md]: Fixed in 28f30f9 — corrected the Files entry to src/content/docs/guide/ and removed the caveat rather than keeping it. HTML regenerated.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/components/astro/Card.astro [hunk 10]
```diff
@@ -31,4 +57,23 @@ const safeCardImage = cardImage
...
+    line-height: 1.7;
+    color: var(--ink-soft);
+    background-color: var(--paper-sunk);
+    border-top: 1px solid var(--rule);
+    /* The snippet is wider than a 320px viewport; it scrolls inside its own box
+       so the page body never scrolls horizontally (WCAG 2.1 SC 1.4.10 Reflow). */
+    overflow-x: auto;
+    white-space: pre;
+  }
+</style>
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/components/astro/Card.astro:85]: Fixed in 28f30f9 — added padding: 1rem 1.25rem to match the hero specimen block.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/components/astro/FeatureCards.astro [hunk 11]
```diff
@@ -37,25 +56,166 @@ if (category) {
 } else {
   features = getTopFeatures(count)
 }
+
+// Clamped so a caller cannot promote more specimens than there are features,
+// and so a negative or fractional count falls back to the default tiering.
+const promotedCount = Math.min(Math.max(0, Math.trunc(promoted) || 0), features.length)
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/components/astro/FeatureCards.astro:64]: Fixed in 28f30f9 — comment now states that fractional counts truncate toward zero, so 2.7 promotes two, and only values truncating to zero or below reach the default path.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: tests/integration/design-tokens.test.ts [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+ * The assertions read the *compiled* stylesheet rather than the SCSS source,
+ * because a token only has a consumer once the cascade Sass emits actually
+ * contains one. `npm run sass:build` writes it compressed (one long line) and
+ * lint-staged then runs Prettier over it, so everything below parses characters
+ * and braces instead of relying on line breaks or indentation.
+ */
+const COMPILED_CSS = fileURLToPath(new URL('../../src/styles/index.css', import.meta.url))
+
+type Declaration = { property: string; value: string }
+type Rule = { selector: string; conditions: string[]; declarations: Declaration[] }
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply tests/integration/design-tokens.test.ts]: Fixed in 28f30f9 — Declaration and Rule properties are now readonly, including the conditions and declarations arrays.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/styles/_design-tokens.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/styles/_design-tokens.scss:393]: Already fixed in ee120c9, pushed before this review ran against 9419654 — same diagnosis and the same remedy you propose (legacy-palette-light mixin included in both :root and the light override). Verified under emulated OS-dark with data-theme=light: --color-neutral-200 and --card-border resolve to #e4e4e7 instead of staying #3f3f46.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/layouts/Layout.astro [hunk 13]
```diff
@@ -1,17 +1,51 @@
...
+const resolvedTitle = pageTitle || 'Authentication'
+const resolvedDescription = pageDescription || 'Sign in to your account'
 ---
 
 <Base
   hideHeader={hideHeader || false}
-  pageTitle={pageTitle || 'Authentication'}
-  pageDescription={pageDescription || 'Sign in to your account'}
+  pageTitle={resolvedTitle}
+  pageDescription={resolvedDescription}
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/layouts/Layout.astro:19]: Not changed — measured, and this is pre-existing rather than introduced here. The Authentication fallback is already in the base commit 52cde11: `pageTitle={pageTitle || Authentication}`. I swapped the pre-PR Layout back in and re-measured /success and /styles/card: both render `<title>Authentication</title>` and `<h1>Authentication</h1>` on 52cde11, byte-identical to this branch. The claim that Base previously rendered an empty h1 does not hold, because Layout already resolved the string before passing it to Base, so `pageTitle || ""` never saw undefined. The leak is real and worth its own fix, but it is outside this PR and fixing it would change routes this PR does not touch.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: tests/components/FeatureCards.astro.test.ts [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+ */
+function countAttr(html: string, name: string): number {
+  return html.match(new RegExp(`\\s${name}(?=[\\s>=])`, 'g'))?.length ?? 0
+}
+
+/** Text content of every `<hN>` element, inner tags stripped and entities left as-is. */
+function headingTexts(html: string, level: number): string[] {
+  const matches = html.matchAll(new RegExp(`<h${level}\\b[^>]*>([\\s\\S]*?)</h${level}>`, 'g'))
+  return [...matches].map(m => m[1].replace(/<[^>]*>/g, '').trim())
+}
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply tests/components/FeatureCards.astro.test.ts:39]: Not changed — tests are outside the type-check scope. tsconfig.json sets `include: ["src/**/*"]`, so tsc never loads tests/ or e2e/; `tsc --listFilesOnly` returns zero matches for this file, and npm run type-check reports 127 errors both before and after, none in these files. noUncheckedIndexedAccess therefore cannot apply here. The regex guarantees the capture group whenever it matches, and the suite passes 20/20 at runtime.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: tests/components/FeatureCards.astro.test.ts [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+    it('emits the import line that reproduces a promoted card', async () => {
+      // The specimen exists to show the source next to the rendered component;
+      // a <pre> holding only the description would satisfy the counts above.
+      const [first] = getTopFeatures(6)
+      const html = await render({ promoted: 1 })
+      const specimen = /<pre\b[^>]*\sdata-card-code[^>]*>([\s\S]*?)<\/pre>/.exec(html)?.[1] ?? ''
+
+      expect(specimen).toContain('#components/astro/Card.astro')
+      expect(specimen).toContain(escapeText(first.title))
+    })
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply tests/components/FeatureCards.astro.test.ts:105]: Not changed — same reason as the consolidated comment on lines 36-39: tsconfig include is `src/**/*`, so this file is never type-checked and noUncheckedIndexedAccess does not reach it.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/components/astro/FeatureCards.astro [hunk 16]
```diff
@@ -14,18 +30,21 @@ export type Props = {
...
 
 const {
   count = 6,
   category,
   showAll = false,
-  sectionTitle = 'Key Features',
+  sectionTitle,
   className = '',
+  promoted = 0,
 } = Astro.props
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/components/astro/FeatureCards.astro:48]: Not changed — pre-existing, and outside this PR. `className` is declared in Props and destructured but never applied in the base commit 52cde11 as well, so this PR neither introduced nor widened the gap. Worth fixing (apply it or drop it from Props), but as its own change rather than a drive-by here.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: tests/integration/design-tokens.test.ts [hunk 17]
```diff
@@ -0,0 +1,225 @@
...
+ * full alias layer that nothing read, so every painted surface fell through to the
+ * @fpkit/acss defaults and the dark-mode block could not change a single pixel.
+ *
+ * The assertions read the *compiled* stylesheet rather than the SCSS source,
+ * because a token only has a consumer once the cascade Sass emits actually
+ * contains one. `npm run sass:build` writes it compressed (one long line) and
+ * lint-staged then runs Prettier over it, so everything below parses characters
+ * and braces instead of relying on line breaks or indentation.
+ */
+const COMPILED_CSS = fileURLToPath(new URL('../../src/styles/index.css', import.meta.url))
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply tests/integration/design-tokens.test.ts:19]: Fixed in 02f0718 — both parts confirmed and both were right. Added `npm run sass:build` to `pretest`. Also promoted this test (and FeatureCards.astro.test.ts) to a blocking CI step: ci.yml ran `vitest run --exclude "tests/integration/**"`, which swept this up with the Supabase-backed tests it was meant to skip, so the gate proving --card-background/--header-background still have painting consumers was not running at all. It needs no credentials — it only reads the compiled stylesheet. Verified the gate is real: with a stale index.css it fails 10 assertions; 20/20 once pretest rebuilds it.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: src/components/astro/HomeHero.astro [hunk 18]
```diff
@@ -0,0 +1,317 @@
...
+ *
+ * The direction is "colour marks hydration": static content is ink on paper,
+ * and the accent (`--island`) appears only on things a visitor can operate --
+ * the two calls to action, the specimen's own link and the focus rings.
+ * Nothing decorative carries it, which keeps the rule machine-checkable.
+ *
+ * The signature element is the specimen: a real `Card` rendered from the same
+ * values the code sample prints, so the sample cannot drift from the component
+ * standing beside it.
+ */
```
[the remark this answers is not in the document]
Comment (shawn-sandy) [reply src/components/astro/HomeHero.astro:36]: Fixed in 02f0718 — added @component and @example, matching Card.astro and FeatureCards.astro. The example shows the required slot="header" (without it the hero renders into the page body and the shared Header stays) plus the three overridable strings.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
